src/test/java/de/meggsimum/w3w/What3WordsTest.java

```diff
@@ -104,4 +157,31 @@ public void testWhat3WordsException() {
 		assertTrue(thrown);
 	}
 
+	public void testNonAsciiCharatersFR() {
```

Comment:
Would you mind adding some little API-doc blocks, like the other test methods have?